build.rs

```diff
@@ -213,45 +214,73 @@ fn main() {
     }
 }
 
-fn make_zlib(compiler: &cc::Tool, src_dir: &path::Path, out_dir: &path::Path) {
-    let src_dir = src_dir.join("zlib");
+fn make_zlib(compiler: &cc::Tool, src_dir: &path::Path, _: &path::Path) {
```

Comment:
If you don't use a previously present argument, can you remove it?

Reply:
All right, deleted.